core/test/oracle/Store.js

```diff
@@ -64,13 +65,30 @@ contract("Store", function(accounts) {
     // TODO Check that only permitted role can change the fee
   });
 
-  it("Final fees", async function() {
+  it.only("Final fees", async function() {
     // Add final fee and confirm
-    await store.setFinalFee(arbitraryTokenAddr, { rawValue: web3.utils.toWei("5", "ether") }, { from: owner });
+    const result = await store.setFinalFee(
+      arbitraryTokenAddr,
+      { rawValue: web3.utils.toWei("5", "ether") },
+      { from: owner }
+    );
+
+    truffleAssert.eventEmitted(result, "NewFinalFee", ev => {
+      return ev.newFinalFee.rawValue === web3.utils.toWei("5", "ether");
+    });
     const fee = await store.computeFinalFee(arbitraryTokenAddr);
     assert.equal(fee.rawValue, web3.utils.toWei("5", "ether"));
   });
 
+  it.only("Weekly delay fees", async function() {
```

Comment:
```suggestion
  it("Weekly delay fees", async function() {
```

Reply:
this was a mistake, sorry! submitted the PR too quickly.